### DISCUSSION TITLE
Cursos presenciais: qual a opinião de vocês?

### DISCUSSION THREAD

**Discussion by joaoricardofl** [frontendbr/forum#1390] [Discussão]
Pessoal,

Levando em conta que cursos presenciais geralmente são mais caros que os cursos online e não têm tanta flexibilidade de horário. E também o acrescimento no currículo, o aproveitamento do curso e a network que um curso desses pode trazer, vocês acham que vale a pena? Ou vocês preferem fazer só cursos online? Vocês veem outros benefícios?

Alguém indica algum curso de React no RJ?

**Comment by renanlbl** [comment]
@joaoricardofl

Cursos presenciais, dependendo o que você vai aprender (por causa da curva de aprendizagem), tem que ser mt longo, visto os horários que normalmente são dispostos.

Vai de pessoa pra pessoa, mas eu prefiro curso online, porque vc pode voltar quantas vezes vc quiser o vídeo, pode reever, pode pausar para pesquisar algo que vc ainda não entende, estudar no horário que você quiser, mas tem que ter mais disciplina pra não procrastinar.

Cursos presenciais normalmente já tem a grade pré-definida, a vantagem é que vc vai ta la com o professor e pode perguntar dele até tu entender afundo, e vc sabendo que pagou caro em um curso presencial, a chance de vc faltar é menor... 

Enfim, essa é minha opinião, e acredito que é tudo questão de adaptação, o ser humano é o animal que se adapta mais fácil a qualquer tipo de situação, hoje em dia vc pode preferir cursos presenciais por questões de costume, mas você forçando um pouco com os cursos onlines, logo se adaptará.

**Comment by gabriel-hahn** [comment]
Sou bastante fã dos cursos onlines. Acredito que te entregam um ensino mais direto e com mais velocidade, desde que tu saiba exatamente o que precisa para avançar na tecnologia que tu quer compreender.
Cursos presenciais possuem o diferencial de um professor para tirar dúvidas específicas, além do network que é MUITO importante, mas costumam ser ligeiramente mais caros.
Acredito que vai depender dos teus objetivos e tempo para estudo. Por exemplo, tu quer aprender mais sobre React. Se tu já possui uma base sólida em desenvolvimento, tem certa experiência, acredito que um curso online seja mais indicado, pois tu já sabe como procurar as respostas para tuas dúvidas sozinho e não vai precisar de tanto "suporte" inicial, além de te entregar um conhecimento mais rápido. Só recomendo pesquisar bem antes de sair fazendo qualquer curso. Existem os muito bons, assim como uns péssimos que podem fazer tu achar que todo curso online daquele assunto é ruim hehe
Sobre currículo, eu posso estar errado, mas tenho a impressão que um dos poucos diplomas que interferem em algo hoje em dia é o da faculdade. Cursos fora disso vão te ajudar e entender uma tecnologia e aplicar ela, mas as empresas estão mais preocupadas em ver teu conhecimento técnico (próprio GitHub por exemplo) do que apenas vários diplomas e certificados.

**Comment by asisanto** [comment]
sim senhor

**Comment by HmmerHead** [comment]
Gosto dos dois. Tenho preferido um pouco mais os presenciais, porém está difícil achar alguma coisa, visto que boa parte dos cursos estão nas cidades grandes, o que complica no meu caso.
Inclusive, pergunto aos que já fizeram cursos presenciais, que deem sua alguma sugestão, e opinião.

Quanto ao curso online confirmo o que foi dito anteriormente. Uma boa pesquisa é essencial para encontrar algo bom, e que valha o investimento.

Direi a minha experiência para outros que estão na dúvida.
Eu assinei a **Alura**. Um bom investimento. Algumas carreiras possuem muitos cursos, outras nem tanto, só o básico até o momento.
**Treina Web** eu peguei por um mês. Só fiz o curso de Laravel. Muito abrangente.
Na **Udemy**, tenho alguns. Lá é importe ver a nota e tudo mais.

**Comment by gabriel-hahn** [comment]
**Udemy** realmente tem bastante cursos bem completos e com bons professores (maioria em Inglês, mas tem a legenda que quebra um galho caso alguém não entenda a língua muito bem). Boa parte dos cursos que faço online estão nesta plataforma. Não me adaptei com outros sites de cursos que seguem a mesma "vibe" da Udemy, como o Alura ou School of Net.
Curto também alguns cursos mais específicos, como o bootcamp GoStack (somente React, Node e React Native) da **Rocketseat**. Não é um curso extenso, mas foca exatamente na estrutura principal das tecnologias utilizadas.

**Comment by didiraja** [comment]
Eu fiz os da Caelum e são muito bons. A grande vantagem é que voce acumula a informação muito mais rápido, principalmente se vc tem problema de disciplina em parar pra estudar online. Porém, em qualidade, está totalmente equiparado com os cursos de Udemy, que na verdade até tem uma vantagem pela variedade.

A propósito, está se formando uma turma de React na Caelum pro mês que vem, aqui no RJ.
